addons/mail/static/src/models/country/country.js

```diff
@@ -16,12 +16,13 @@ function factory(dependencies) {
          * @override
          */
         static _createRecordLocalId(data) {
-            return `${this.modelName}_${data.id}`;
+            return `${this.modelName}_${data.id ? data.id : data.code}`;
```

Comment:
I saw a discussion about this but maybe it was on another PR because I can't find it anymore.

Anyway I think this is a bad idea. localId is used to uniquely identify a record, and if the same country is "inserted"/"find" once by `id` and then once by `code`, it will fail to do reconciliation them and it will exist 2 times.

Reply:
The best approach is to ensure the `id` is given in all situations, if possible (I didn't look at the issue here in detail).

Reply:
That's what I discussed with @alexkuhn and the conclusion was "it's better to have possibly duplicated data than no data" but yeah we could add a key if it's ok to add keys in response

Reply:
If adding `id` to response is enough to resolve this then for sure add it.